src/app/index.css

```
@@ -181,24 +189,28 @@
-  padding-bottom: var(--safe-area-inset-bottom);
-  padding-left:   var(--safe-area-inset-left-legacy);
-  padding-left:   var(--safe-area-inset-left);
-
-  /* 선택사항: 배경색 지정 시 뷰포트 끝까지 채우기 */
-  background-color: #fff;
-  min-height: 100vh;
-  box-sizing: border-box;
+:root {
+  /* 구형 사파리 호환용 */
+  --safe-top:    constant(safe-area-inset-top);
+  --safe-bottom: constant(safe-area-inset-bottom);
```

Biome (1.9.4)

[error] Unexpected unknown function: constant

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)